thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -305,7 +308,7 @@ export function buildAnomalyStats(alertEvalMetrics, mode, severity = '30', isPer
     let newData = alertEvalMetrics.projected[stat.key];
     let isPercentageMetric = stat.units === '%';
     let isTotal = stat.key === 'totalAlerts';
-    stat.showProjected = mode === 'explore';
+    stat.showProjected = false;
```

Comment:
Let's add a comment that this is temporary + JIRA Ticket tracking the status of the fix

Reply:
ok